eZ/Publish/Core/Persistence/Cache/Tests/ContentHandlerTest.php

```diff
@@ -346,7 +346,7 @@ public function testLoadVersionInfoCacheIsMiss()
             ->with();
 
         $handler = $this->persistenceCacheHandler->contentHandler();
-        $handler->loadVersionInfo(2, 1);
+        $handler->loadVersionInfo(2, null);
```

Comment:
Is the case with the given `versionNo` covered elsewhere?

Reply:
I should duplicate the test to cover both ideally.

Reply:
Added